### PR TITLE
Switch color icons to imageHtml

### PR DESCRIPTION
**Pull request by tk**
Switching from imageUrl to imageHtml for the color icons.

This is necessary to show them in the vs-code-bpmn-io extension, that has a stricter ContentSecurityPolicy for inlined image sources and styles.

### CODE REVIEW

**Review by smbea — CHANGES_REQUESTED** [review]
The color icons cut off a little bit, see:
Screenshot 2023-10-30 at 09 11 59

File: colors/ColorPopupProvider.js [hunk 1]
```diff
@@ -54,21 +50,21 @@ ColorPopupProvider.$inject = [
 ColorPopupProvider.prototype.getEntries = function(elements) {
   var self = this;
 
-  var colorIcon = domify(`
-    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" height="100%">
-      <rect rx="2" x="1" y="1" width="22" height="22" fill="var(--fill-color)" stroke="var(--stroke-color)" style="stroke-width:2"></rect>
+  var colorIconHtml = `
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" height="100%" width="100%">
+      <rect rx="2" x="1" y="1" width="20" height="20" fill="var(--fill-color)" stroke="var(--stroke-color)" style="stroke-width:2"></rect>
     </svg>
```
Comment (smbea) [inline colors/ColorPopupProvider.js:56]: Why the change of width from `22` to `20`? It causes the icons to be a bit smaller:

Screenshot 2023-10-31 at 14 05 19

I tested with `22` and it seems to work as expected.

Comment (tk) [reply colors/ColorPopupProvider.js:56]: The parent element constraints the icon to 1em (which is less than 20px on some high resolution monitors), which leads to the cut off effect mentioned in the first review when the browser shows the website with 100%. If you zoom in the cut off effect is gone.
I can change it back to 22, but then the cut off effect will be there again.

**Comment by tk** [comment]
@smbea : I experimented and found a setting that hopefully solves both issues.

**Review by smbea — APPROVED** [review]
I think it looks good on the latest changes.
Thanks for your contribution!

**Comment by smbea** [comment]
I will follow up with releasing this

**Pull request merged**